glean-core/android/src/main/java/mozilla/telemetry/glean/private/TimingDistributionMetricType.kt

```diff
@@ -74,3 +88,32 @@ class TimingDistributionMetricType(meta: CommonMetricData, timeUnit: TimeUnit) :
     @VisibleForTesting(otherwise = VisibleForTesting.NONE)
     fun testGetNumRecordedErrors(error: ErrorType) = inner.testGetNumRecordedErrors(error)
 }
+
+class LocalTimingDistribution(
+    val metric: TimingDistributionMetricType,
+    val inner: FfiLocalTimingDistribution,
+) : AutoCloseable {
```

Comment:
Might want `protected fun finalize() { /* TODO: bug 1691073 */ report_some_error(); ffi_abandon_buffer(this);` or whatever to help report misuse and prevent memory leaks.